code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -267,9 +267,18 @@
 								mob_is_hit = TRUE
 								break
 				if(mob_is_hit)
+					if(ammo.flags_ammo_behavior & AMMO_SHRAPNEL)
+						if(world.time > L.last_shrapnel)
+							L.shrapnel_counter = 0
+						L.shrapnel_counter++
+						L.last_shrapnel = world.time + LIVING_LAST_SHRAPNEL_MOD
+						if(L.shrapnel_counter > LIVING_MAX_SHRAPNEL)
+							return FALSE
```

Comment:
Only if the bullet has the Shrapnel munition flag... and there are plenty of flag specific behaviours throughout the file.

Reply:
Doesn't mean it's good practice. This special snowflakery makes no sense, can actually create really odd situations, and is basically lazy balancing. I've given you other solutions.

Reply:
You seem to have misunderstood. I say reduce accuracy in general, so you don't have high chances of being skewered by 50 AP flechette projectiles. AP could really get lowered here, by the byes. That is one option.

The other is to try to emulate the explosion going in different directions, as the grenade and the mob would only occupy the same space if it was inside it. Since that shouldn't be the case, you can move the source of the shot one tile for some sharpnels, and better represent the situation.

These are not the only two possibilities, mind you.

Reply:
I didn't misunderstand anything.

1: Reducing the accuracy isn't going to do spit for limiting ground zero/direct hit damage which is what we're actually concerned about, unless we make the accuracy so atrocious that it about completely fails to hit things beyond the first tile.

2: I'm not sure how arbitrarily moving the origin square of some tiles more accurately represents the dispersal pattern of the shrapnel/explosion. As is, the dispersal pattern is quite representative given that it features on the whole a fairly even distribution of projectiles. You should probably run the code to see it for yourself.

Reply:
1: Spooky suggested accuracy working differently here. Food for thought.

2: And it works perfectly until you put the mob in the same tile. Which is what we are discussing about. Because we don't yet use a pixel-based system, but tile-based.

Reply:
1: He suggested inverted accuracy, where it's less accurate the closer you are which while better than shafting accuracy on the basis of the number or projectiles is still likely to be problematic (particularly in the event where you're close to but not at ground zero, so you benefit from bad accuracy + dispersal), and is certainly no less arbitrary than simply capping the damage.

2: Yes it does work perfectly, and being at ground zero _should_ devastate you. The problem is that it devastates too much. I don't think arbitrarily changing origin squares keyed to the number of projectiles and thus making the effect much less consistent rather than applying a damage cap is the better option.

Reply:
Personally I'd add a larger number of shrapnel with lower AP and a low hit chance each. This is not an aimed shot.

Reply:
More shrapnel = more resources consumed though, and it doesn't obviate the need for capping.